pkgs/applications/misc/gcfflasher/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, pkgs
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcfflasher";
+  version = "4.0.3-beta";
+
+  src = fetchFromGitHub {
+    owner = "dresden-elektronik";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-m+iDBfsHo+PLYd3K8JaKwhIXcnj+Q8w7gIgmHp+0plk=";
+  };
+
+  nativeBuildInputs = with pkgs; [ gcc pkg-config libgpiod ];
+
+  buildPhase = ''
+    ./build_posix.sh
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv GCFFlasher $out/bin
+  '';
```

Comment:
```suggestion
```

Ideally this is redundant, once you use cmake to build. Also, please note that when you overwrite a phase, a thing you rarely need to do, you should explicitly runs its pre and post hooks.